docs/contribution_guide/ways_to_contribute.rst

```diff
@@ -0,0 +1,17 @@
+Ways to contribute
+==================
+
+There are many different ways to contribute to the Fairlearn community, for
+example:
+
+* contributing code to add/enhance
+    * unfairness mitigation techniques
+    * fairness metrics
+    * visualizations 
+* writing example notebooks
+* improving documentation
+    * in the code through docstrings
+    * for the website through user guides, etc.
+* reporting bugs
+* requesting new features
+* writing proposals for larger changes
```

Comment:
The styling of the above list looks off. I noticed two things:
* you jump two levels for sublists
* you don't enter empty lines around sublits: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks

Reply:
Yes, well spotted! Lots of little things that are different from markdown :-) Should be better now